pages/docs/features/inngest-functions/error-retries/failure-handlers.mdx

```diff
@@ -2,12 +2,21 @@ import { CardGroup, Card, CodeGroup, Callout } from "src/shared/Docs/mdx";
 
 # Failure handlers
 
-If your function exhausts all of its retries, it will be marked as "Failed." You can handle this circumstance by providing an [`onFailure`](/docs/reference/functions/handling-failures) handler when defining your function.
+If your function exhausts all of its retries, it will be marked as "Failed." You can handle this circumstance by either providing an [`onFailure/on_failure`](/docs/reference/functions/handling-failures) handler when defining your function, or by listening for the [`inngest/function.failed`](/docs/reference/system-events/inngest-function-failed) system event.
+
+The first approach is function-specific, while the second covers all function failures in a given Inngest environment.
+
+<Callout>
+  In any way you will need to re-sync your app for the onFailure-code to register and work!
+</Callout>
```

Comment:
```suggestion
```

This notice is not necessary as it applies to all documentation pages suggesting code changes.

Reply:
It was non obvious to me and i spent quite a bit of time debugging. It fails silently, so I'd advocate for a hint. It will save people time

Reply:
Understood, then we'll add a callout in the "Deployment" part instead